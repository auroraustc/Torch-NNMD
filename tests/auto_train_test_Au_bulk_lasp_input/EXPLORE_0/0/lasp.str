!BIOSYM archive 3
PBC=ON
Material Studio CAR format file
!DATE     Sep 08 10:57:38 2019
PBC    2.8001    2.8001    6.8589   90.0000  120.0000  120.0000
Au     0.000000000     0.000000000     0.000000000 XXXX 1      xx      Au  0.000
Au    -0.000000000     1.616663230     2.286307066 XXXX 1      xx      Au  0.000
Au     1.400071427     0.808331615     4.572614132 XXXX 1      xx      Au  0.000
end
end
