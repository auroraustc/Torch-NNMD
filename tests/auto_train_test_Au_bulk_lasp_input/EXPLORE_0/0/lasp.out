+--------------------------------------------------------------------------------+
|                 Welcome to Fantastic Atomic World in Neural Network            |
|                              ---------                                         |
|                               L A S P                                          |
|                              ---------                                         |
|           Large-scale Atomic Simulation with neural network Potential          |
|                             Version 2.0.1     Feb. 2019                        |
|                                Shanghai, China                                 |
|                   More on Website: www.lasphub.com                             |
|                                                                                |
|      Please cite: S.D. Huang, C. Shang, P.L. Kang, X.J. Zhang and Z.P. Liu ,   |
|                   "LASP: Fast Global Potential Energy Surface Exploration" ,   |
|                   WIREs. Comp. Mole. Sci. 2019, (DOI: 10.1002/wcms.1415)       |
|                                                                                |
|                  Compiled by IntelFortran version: ifort      12               |
|                  CPU Parallel Run on    1      cores                           |
+--------------------------------------------------------------------------------+
 
+--------------------------------------------------------------------------------+
+   The PES is calculated by 
+   external
+--------------------------------------------------------------------------------+
 
|********************************************************************************|
|                             Welcome to SSW package                             |
|               Stochastic Surface Walking Global Optimization 2.0               |
|                     ----  now as a part of LASP program                        |
|                        Fudan University Liu group 2013-2018                    |
|               Developers: Cheng Shang, Xiaojie Zhang, Zhipan Liu               |
|********************************************************************************|
 
         SSW Job Start at 11:58:15  8th September  2019                               
|   System size: Number of atoms         3
|   Fix Cell Structure Search: Run_type=5
|   Key Parameters Utilized From Input: 
|   Number of Gaussian for atomic move         9
|   Monte Carlo Temperature    500.00
|   Gaussian ds for each atomic move      0.60
|   Random mode Level         0
|   Maximum Localmode-to-Globalmode ratio        50
|   Maximum SSW steps        10
|   --Please also check allkey.log for SSW Parameters    
|                                                                
|  SSW cycles start                                               
|********************************************************************************|
|********************************************************************************|
Str symm and Q       0     0 longQ T           -8.901066   Dih      1.1106      0.9741      0.9436
 ----------------------
 NewStart                      NewStart                      
 ----------------------
Energy,force          -8.901066000       0.034155000
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.901066000       0.034155000
LBFGS opt       0.034155      -8.901066       0
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.901091000       0.034129000
LBFGS opt       0.034129      -8.901091       1
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.906261000       0.014966000
LBFGS opt       0.014966      -8.906261       2
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.906857000       0.012736000
LBFGS opt       0.012736      -8.906857       3
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.907455000       0.012079000
LBFGS opt       0.012079      -8.907455       4
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909412000       0.016966000
LBFGS opt       0.016966      -8.909412       5
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909420000       0.016974000
LBFGS opt       0.016974      -8.909420       6
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909943000       0.017788000
LBFGS opt       0.017788      -8.909943       7
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909950000       0.017806000
LBFGS opt       0.017806      -8.909950       8
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909956000       0.017823000
LBFGS opt       0.017823      -8.909956       9
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909963000       0.017841000
LBFGS opt       0.017841      -8.909963      10
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909969000       0.017858000
LBFGS opt       0.017858      -8.909969      11
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909977000       0.017876000
LBFGS opt       0.017876      -8.909977      12
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909985000       0.017893000
LBFGS opt       0.017893      -8.909985      13
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909990000       0.017911000
LBFGS opt       0.017911      -8.909990      14
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.909998000       0.017929000
LBFGS opt       0.017929      -8.909998      15
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910005000       0.017946000
LBFGS opt       0.017946      -8.910005      16
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910011000       0.017964000
LBFGS opt       0.017964      -8.910011      17
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910019000       0.017982000
LBFGS opt       0.017982      -8.910019      18
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910026000       0.018000000
LBFGS opt       0.018000      -8.910026      19
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910032000       0.018018000
LBFGS opt       0.018018      -8.910032      20
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910040000       0.018036000
LBFGS opt       0.018036      -8.910040      21
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910047000       0.018054000
LBFGS opt       0.018054      -8.910047      22
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910054000       0.018071000
LBFGS opt       0.018071      -8.910054      23
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910062000       0.018089000
LBFGS opt       0.018089      -8.910062      24
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910069000       0.018108000
LBFGS opt       0.018108      -8.910069      25
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910076000       0.018126000
LBFGS opt       0.018126      -8.910076      26
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910082000       0.018144000
LBFGS opt       0.018144      -8.910082      27
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910089000       0.018162000
LBFGS opt       0.018162      -8.910089      28
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910097000       0.018180000
LBFGS opt       0.018180      -8.910097      29
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910105000       0.018198000
LBFGS opt       0.018198      -8.910105      30
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910111000       0.018216000
LBFGS opt       0.018216      -8.910111      31
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910118000       0.018235000
LBFGS opt       0.018235      -8.910118      32
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910126000       0.018253000
LBFGS opt       0.018253      -8.910126      33
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910132000       0.018271000
LBFGS opt       0.018271      -8.910132      34
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910140000       0.018289000
LBFGS opt       0.018289      -8.910140      35
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910147000       0.018308000
LBFGS opt       0.018308      -8.910147      36
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910154000       0.018326000
LBFGS opt       0.018326      -8.910154      37
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910162000       0.018345000
LBFGS opt       0.018345      -8.910162      38
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910169000       0.018363000
LBFGS opt       0.018363      -8.910169      39
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910175000       0.018382000
LBFGS opt       0.018382      -8.910175      40
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910183000       0.018400000
LBFGS opt       0.018400      -8.910183      41
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910191000       0.018419000
LBFGS opt       0.018419      -8.910191      42
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910197000       0.018437000
LBFGS opt       0.018437      -8.910197      43
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910205000       0.018456000
LBFGS opt       0.018456      -8.910205      44
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910212000       0.018474000
LBFGS opt       0.018474      -8.910212      45
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910219000       0.018493000
LBFGS opt       0.018493      -8.910219      46
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910227000       0.018512000
LBFGS opt       0.018512      -8.910227      47
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910233000       0.018530000
LBFGS opt       0.018530      -8.910233      48
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910241000       0.018549000
LBFGS opt       0.018549      -8.910241      49
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910249000       0.018568000
LBFGS opt       0.018568      -8.910249      50
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910255000       0.018587000
LBFGS opt       0.018587      -8.910255      51
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910263000       0.018605000
LBFGS opt       0.018605      -8.910263      52
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910271000       0.018624000
LBFGS opt       0.018624      -8.910271      53
 ----------------------
 Allopt                        bfgs_opt                      
 ----------------------
Energy,force          -8.910277000       0.018643000
LBFGS opt       0.018643      -8.910277      54
